code-challenges.test.js

```diff
@@ -49,9 +101,39 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+describe("wordsCapitilized", () => {
+  it("takes in an array of words and returns an array with all the words capitalized", () => {
+    expect(wordsCapitilized(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(wordsCapitilized(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
+
+// ● wordsCapitilized › takes in an array of words and returns an array with all the words capitalized
+//
+//   ReferenceError: wordsCapitilized is not defined
+
 
 // b) Create the function that makes the test pass.
 
+//Create a function - wordsCapitilized
+//Parameter- array
+//Create a new variable within function to take in data
+//Use an interation method .map to go through the array
+//Return the element, use .charAt() mehtod to start at the 0 index of each string in the array
+//Use the .toUpperCase() method to capitilize
+//Use .substring() method starting at index 1, then adding .toLowerCase() method to convert all elements after the first element back to lower case
+//Return strings back into an array
+
+const wordsCapitilized = (array) => {
+  let words = array.map(element => {
+  return element.charAt(0).toUpperCase() + element.substring(1).toLowerCase()
+  })
+  return words.splice("")
```

Comment:
In this case, .splice is not actually doing anything and could be removed.  Just simply returning words would be perfect.  I really like that you also used .toLowerCase() on the substring as this is thinking about edge cases and how to handle them.  Awesome job!